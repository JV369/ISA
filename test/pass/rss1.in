./feedreader http://www.mocky.io/v2/5bd585de310000600041db02
