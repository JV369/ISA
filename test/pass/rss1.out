*** TestOfRSS ***
Test RSS2
Test RSS1

